main/webserver_v2_main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"fault-tolerance/config"
+	"net/http"
+	"net/http/httputil"
+	"log"
+	"fault-tolerance/scheduler"
+	"fault-tolerance/ping"
+)
+
+func NewMultipleHostReverseProxy(scheduler *ping.Scheduler) *httputil.ReverseProxy {
+	director := func(req *http.Request) {
+		backEnd, err := scheduler.GetBackend()
+		if err != nil {
+			fmt.Printf("%v\n", err)
```

Comment:
need not, because this function does not return anything. function just modifies backend url and backend scheme. If there is not backend, then backend url will default to load-balancer url, and the request fails with 404.